pkg/customers/address.go

```diff
@@ -90,13 +119,20 @@ func updateCustomerAddress(logger log.Logger, repo CustomerRepository) http.Hand
 			return
 		}
 
+		ownerID := customerID
+		var representativeID string
+		if ownerType == client.OWNERTYPE_REPRESENTATIVE {
+			representativeID = route.GetRepresentativeID(w, r)
```

Comment:
If representativeID isn't present this will have written an error to `w` (`moovhttp.Problem(w, ErrNoRepresentativeID)`). I think it needs
```go
if representativeID == "" {
    return
}
```